pagopa/io-app
# fix(IT Wallet): [SIW-1754] Missing contextual help inside identification mode screen

## Short description
This pull request includes a small change to the `ItwIdentificationModeSelectionScreen` component. The change adds a new property to display a help button in the header.

## List of changes proposed in this pull request
- Added `showHelp: true` inside ItwIdentificationModeSelectionScreen

## How to test
Start the flow to obtain eID, navigate to the login page (SPID, CIE login), and click the `?` on the header. The button should be visible and pressable
